python/packages/azure-ai/agent_framework_azure_ai/_chat_client.py

```diff
@@ -161,6 +161,7 @@ def __init__(
         project_endpoint: str | None = None,
         model_deployment_name: str | None = None,
         async_credential: AsyncTokenCredential | None = None,
+        should_cleanup_agent: bool = True,
```

Comment:
Looking good.
Should we add a note to relevant chatclient samples mentioning the availability of this flag? 
This way, people following the samples know that its there and that they have options to handle the "dissappearing agent problem"